GUI/src/cz/fidentis/gui/Installer.java

```diff
@@ -72,6 +72,13 @@ public void restored() {
          UIManager.put("text", new Color(0, 0, 0));}*/ catch (URISyntaxException | ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException | ConcurrentModificationException ex) {
             Exceptions.printStackTrace(ex);
         }
+        
+        // delete the temp folder if it existed before the start of the program
+        try {
+            FileUtils.instance().createTMPfolder(true);
```

Comment:
nepodarilo sa teda prist na to ako na splash screen prihodit spravu? Skus toto ci to pomoze https://forums.netbeans.org/ntopic23810.html